R/locations.R

```diff
@@ -12,6 +12,10 @@
 #' @template radius
 #' @template limit
 #' @template page
+#' @template siteType
+#' @template activationDate
+#' @template completeness
```

Comment:
there should be two params in the docs as in the function (completeness_from and completeness_to)

Reply:
after that `devtools::document()` should be run.

Reply:
Oh, yes.. But this is true for activationDate and inletHeight as well right?